.eslintrc.js

```diff
@@ -33,7 +35,16 @@ module.exports = {
     ],
     'arrow-body-style': ['error', 'always'],
     'import/extensions': ['error', 'always'],
-    strict: ['error', 'global']
+    strict: ['error', 'global'],
+    'jsdoc/require-description': 'warn',
+    'jsdoc/require-param': 'warn',
+    'jsdoc/require-returns': 'warn',
+    'jsdoc/check-tag-names': 'warn',
+    'jsdoc/check-alignment': 'warn',
+    'jsdoc/newline-after-description': 'off', // does not work with 'use strict'
+    'jsdoc/check-indentation': 'warn',
+    'jsdoc/lines-before-block': 'warn',
+    'jsdoc/check-types': 'warn'
```

Comment:
This rule alone finds 744 problems. 

```
 Invalid JSDoc @param "licenceName" type "String"; prefer: "string" 
 Invalid JSDoc @returns type "Object"; prefer: "object"
Invalid JSDoc @param "index" type "Number"; prefer: "number" 
```

We can discuss the is this rule is needed or not.